features/issues/components/issue-filters/issue-filters.tsx

```diff
@@ -0,0 +1,102 @@
+import { useState, useRef, ChangeEvent } from "react";
+import { Select } from "@features/ui";
+import { Input, InputIcon } from "@features/ui";
+import { Button, ButtonSize, ButtonIcon } from "@features/ui";
+import { statusOptions, levelOptions } from "./options";
+import { useRouter } from "next/router";
+import { debounce } from "lodash";
+import styles from "./issue-filters.module.scss";
+
+export function IssueFilters() {
+  const router = useRouter();
+
+  const [filters, setFilters] = useState({
+    status: router.query.status || "",
+    level: router.query.level || "",
```

Comment:
These two don't need to be kept in a state but can be directly read and updated from the URL query params